Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -6152,34 +6152,43 @@ def com_google_fonts_check_metadata_category_hint(family_metadata):
 @check(
     id = "com.google.fonts/check/colorfont_tables",
     rationale = """
-        No one color font format supports all major user agents, but the combination
-        of COLR & SVG tables is pretty good.
+        Colr v0 fonts are widely supported in most browsers so they do not require
+        an SVG color table. However, Colr v1 is only well supported in Chrome so
+        we need to add an SVG table to these fonts.
 
-        A smart server could prune away one or the other based on user agent,
-        and a dumb server will at least have something that works.
-
-        Fonts that do not pass this check can be fixed with the maximum_color tool
-        available at https://github.com/googlefonts/nanoemoji
+        To add an SVG table, run the maximum_color tool in Nano Emoji,
+        https://github.com/googlefonts/nanoemoji
     """,
     proposal = 'https://github.com/googlefonts/fontbakery/issues/3886'
 )
 def com_google_fonts_check_colorfont_tables(ttFont):
-    """Fonts must have neither or both the tables 'COLR' and 'SVG '."""
-    SUGGESTED_FIX = ("To fix this, please run the font through the maximum_color tool"
-                     " that installs as part of the nanoemoji package"
-                     " (https://github.com/googlefonts/nanoemoji)")
-    if 'COLR' in ttFont.keys() and 'SVG ' not in ttFont.keys():
-        yield FAIL,\
-              Message('missing-table',
-                      "This is a color font (it has a 'COLR' table)"
-                      " but it lacks an 'SVG ' table. " + SUGGESTED_FIX)
-    elif 'COLR' not in ttFont.keys() and 'SVG ' in ttFont.keys():
-        yield FAIL,\
-              Message('missing-table',
-                      "This is a color font (it has a 'SVG ' table)"
-                      " but it lacks an 'COLR' table. " + SUGGESTED_FIX)
-    else:
-        yield PASS, "Looks good!"
+    """Check font has the expected color font tables"""
+    if "COLR" in ttFont:
+        colr_table = ttFont["COLR"]
+        if colr_table.version == 0 and "SVG " in ttFont:
+            yield FAIL, Message(
+                "drop-svg",
+                "Font has a COLR v0 table, which is already widely supported, "
+                "so the SVG table isn't needed."
```

Comment:
Adobe CC does not support COLR table (and probably will not do any time soon, given the Adobe’s history with SVG table), so if a font for GF want to also support Adobe CC, do you want to force it to ship to versions?

Reply:
hm fair enough.. I would demote that check to a WARNING, saying that COLRv0-only is OK for browser use, unless you really need SVG support for desktop apps that don't support COLR. They probably want to keep things simple and not allow too many options when onboarding new color fonts. /cc @rsheeter

Reply:
@khaledhosny maybe file a separate issue so we can discuss there